tests/fixtures/security-check.php

```diff
@@ -15,7 +15,6 @@
     isset($_SERVER['HTTP_CLIENT_IP'])
     || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
     || '127.0.0.1' !== $_SERVER['REMOTE_ADDR']
-    || PHP_SAPI !== 'cli-server'
```

Comment:
that doesn't work with symfony-cli anymore, since it fpm and not `cli-server` then

Reply:
That's a bit annoying. Does Symfony CLI sets an env var or something like this allowing to detect it? /cc @fabpot

Reply:
`strpos($_SERVER['SERVER_SOFTWARE'], 'Symfony Local Server')` might work, WDYT?